### ISSUE TITLE
199 subscriptions only viewable?

### ISSUE THREAD

**Issue by Allanbuzzy** [Wassup789/YouTube-Notifications#14]
I want to import every single subscription I have. I have more than 1000+ subs, however, when I import them, only 199 subscriptions are only showed. I want about 1,034 channels to have notifications. But I can't believe it only shows that limit. Can I ask, is this an issue or is that just there because thats the limit.

**Comment by Wassup789** [comment]
No, there is no limit to the amount of channels you can import, are you using the right Google account? I know there are usually two accounts, the Google+ one and the YouTube one.

**Comment by Classy-Cube** [comment]
I have 454 subscriptions, importing gave me a warning "Importing 197 total channel(s)".
![missing-subscriptions](https://user-images.githubusercontent.com/13388890/32381941-5e020520-c08a-11e7-8f89-54381bd3f6da.GIF)
It didn't import all my subscriptions such as "A Closer Look"
![a_closer_look](https://user-images.githubusercontent.com/13388890/32382120-d3d8ccd4-c08a-11e7-82e0-60b7428b778b.GIF)
Chrome Version 62.0.3202.75 (Official Build) (64-bit) Only extension enabled is YouTube Notifications.
I've tested this a couple times and it's consistently the same unsubscribed channels.
I haven't "Created a channel" for the YouTube account but I doubt that's the issue.
Gifs created using Snagit.
